### ISSUE TITLE
Reproducible builds

### ISSUE THREAD

**Issue by xaur** [decred/dcrdevdocs#62]
Back in November the developers were [testing](https://matrix.to/#/!HEeJkbPRpAqgAwhXWO:decred.org/$157308464734953aRCth:decred.org) reproducible builds with Go 1.13. It would be nice to have instructions about verifying those builds compiled somewhere. I think devdocs is a better fit compared to docs, because building requires the toolchain installed.

**Comment by raedah** [comment]
bump

**Comment by jholdstock** [comment]
Suggested outline for the page:

---
- What reproducible builds are
- Why they are desirable
- How they are achieved
    - Code is currently here https://github.com/jrick/release - Might move to a decred org repo at some point?
- How to verify released build matches source code
---

Any thing missing here @raedah @xaur? And also tagging @jrick as I believe he did most of the implementation on this - anything else noteworthy?

**Comment by jrick** [comment]
that should cover it.  also i have no issues moving the tool under the decred org, but release time is always busy and moving the code around is a bit of a low priority task.

**Comment by jholdstock** [comment]
Once this page is done it should probably be linked from/to https://docs.decred.org/advanced/verifying-binaries/

**Comment by matthawkins90** [comment]
This issue seems to require more progress on decredcommunity/issues#64 first.

**Comment by jrick** [comment]
fyi: https://github.com/decred/release

we moved the repo under the decred org.
